src/installer/tests/Microsoft.NET.HostModel.Tests/Helpers/BundleHelper.cs

```diff
@@ -83,7 +83,7 @@ public static string[] GetExtractedFiles(TestProjectFixture fixture, BundleOptio
         public static string[] GetFilesNeverExtracted(TestProjectFixture fixture)
         {
             string appBaseName = GetAppBaseName(fixture);
-            return new string[] { $"{appBaseName}",
+            return new string[] { $"{appBaseName}.dll",
                                   $"{appBaseName}.exe",
                                   $"{appBaseName}.pdb",
                                   $"{appBaseName}.runtimeconfig.dev.json",
```

Comment:
We need the `{appBaseName}` (without extension) since that's the app's executable on non-Windows (I wonder how come this doesn't fail).